descriptions/ghes-2.18/dereferenced/ghes-2.18.deref.json

```diff
@@ -239831,6 +239831,9 @@
                       "body_text": {
                         "type": "string"
                       },
+                      "mentions_count": {
+                        "type": "integer"
+                      },
```

Comment:
ping @MylesBorins to double check if this key was meant to land in GHES/GHAE yet

Reply:
This came from my PR: https://github.com/github/github/pull/185141
This feature does not exist in GHES/GHAE yet, although I assume it will get there eventually.  I'm unclear what I did wrong or how to fix this though.  So I'm looking for guidance!

Reply:
this has been resolved in the most recent pull request #479